src/routes/LandingPage/SponsorSection/SponsorComponents/SponsorAccentureImage.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import '../index.scss';
+import Acc from 'assets/SponsorImages/accenture-1.png';
+
+const Accenture = () => {
+  return (
+    <div>
+      <img className="logo" src={Acc} alt="accenture logo" />
```

Comment:
I think perhaps this className could be a little more specific  like `sponsor-logo` just to disambiguate it from our own logo.
I like that stuff is containerised but, since all of these images follow almost the same format, we could have a general `<SponsorImage />` component that takes in an image as a prop; so something like 

```jsx
const SponsorLogo = ( {logoSource, altText} ) => {
  return (
    <div className="sponsor-logo-cont">
      <img className="sponsor-logo" src={logoSource} alt={altText} />
    </div>
  );
};
```